modules/n1ql/pages/n1ql-language-reference/stringfun.adoc

```diff
@@ -1,5 +1,6 @@
 = String Functions
 :page-topic-type: concept
+:page-aliases: n1ql:n1ql-language-reference/string-functions
```

Comment:
Doesn't this only need to be fixed in the 4.6 branch? https://docs.google.com/spreadsheets/d/1Kr_qOWdBd-12ak-HenRaX6-Z4X3rqNPk3B9CI-my3Ps/edit#gid=24988422&range=H1038

Reply:
Removed redirect in the 6.0 branch

Reply:
Wait a second, it's back

Reply:
Okay, removed redirect in the 6.0 branch again. Some kind of merge conflict there.